packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -52,43 +51,60 @@ export function SignIn() {
       }}
       onChange={handleChange}
     >
-      <Heading level={1}>{I18n.get('Sign in to your account')}</Heading>
+      <Flex direction="column">
+        <Heading level={3}>{I18n.get('Sign in to your account')}</Heading>
 
-      <FederatedSignIn />
-
-      <Fieldset disabled={isPending}>
-        <UserNameAlias data-amplify-usernamealias />
-
-        <Label data-amplify-password>
-          <Text>{I18n.get('Password')}</Text>
-          <Input name="password" required type="password" />
-          <Box>
-            <Text>{I18n.get('Forgot your password? ')}</Text>
-            <Button
-              onClick={() => send({ type: 'RESET_PASSWORD' })}
-              type="button"
-            >
-              {I18n.get('Reset password')}
-            </Button>
-          </Box>
-        </Label>
-      </Fieldset>
+        <FieldGroup disabled={isPending} direction="column">
+          <UserNameAlias data-amplify-usernamealias />
+          <PasswordField
+            className="password-field"
+            placeholder={I18n.get('Password')}
+            required
+            name="password"
+          />
+        </FieldGroup>
 
-      <Footer>
-        <Text>{I18n.get('No account? ')}</Text>
-        <Button onClick={() => send({ type: 'SIGN_UP' })} type="button">
-          {I18n.get('Create account')}
+        <Button
+          borderRadius={0}
+          isDisabled={isPending}
+          isFullWidth={true}
+          type="submit"
+          variation="primary"
+          isLoading={isPending}
+          loadingText={I18n.get('Signing in')}
```

Comment:
This will need a translation I'm guessing...

Reply:
Yep!